EIPS/eip-5792.md

```diff
@@ -12,61 +12,23 @@ created: 2022-10-17
 
 ## Abstract
 
-Defines new JSON-RPC methods which enable apps to ask a wallet to process a batch of onchain write calls and to check on the status of those calls. Calls can be specified to make use of various wallet-defined capabilities, and there is also a new RPC to enable apps to discover which capabilities a wallet supports. 
+Defines new JSON-RPC methods which enable apps to ask a wallet to process a batch of onchain write calls and to check on the status of those calls. Dapps can specify that these onchain calls be executed taking advantage of specific capabilities previously expressed by the wallet; an additional, novel wallet RPC is defined to enable apps to query the wallet for those capabilities.
```

Comment:
Can we explicitly call out the method (feel like a reader just wants to get straight to the point instead of reading too much preamble)?

```suggestion
Defines a new set of JSON-RPC methods that enable Wallet Consumers to request to send bundles of onchain write calls: 

- `wallet_sendCalls`: a method to send a bundle of onchain write calls that returns a reference identifier.
- `wallet_getCallsReceipt`: a method to extract receipts from the sent calls.
- `wallet_getCapabilities`: a method to discover capabilities that a wallet supports.
```

Reply:
```suggestion
Defines a new set of JSON-RPC methods that enable Wallet Consumers to request to send bundles of onchain write calls: 
- `wallet_sendCalls`: a method to send a bundle of onchain write calls that returns a reference identifier.
- `wallet_getCallsReceipt`: a method to extract receipts from the sent calls.
- `wallet_getCapabilities`: a method to request capabilities that a wallet supports.
```
I love it, with only one word changed to honor my eccentric steelman proclivities changed, i'd love it even more (see last line)

Reply:
Does "request" sound like is asynchronous? As in, you need to switch to wallet context (which you don't)?

Reply:
oh, sorry, I wasn't thinking in terms of runtime sync semantics, I was just thinking about the nitpicky distinction between requesting a capability (which is both feature and permission to use it) and discovering a feature.  the hair-splitting has to do with the semantics of NULL responses (the wallet might SUPPORT the feature but still send back no capability for permission or trust-level reasons, as Dan keeps saying).

Reply:
"discover permitted capabilities"?

Reply:
"query for permitted capabilities"?

Reply:
(if it's this hard to wordsmith maybe it's not a distinction worth making!)

Reply:
anything but "request" :)